### PR TITLE
Add 08early-pycht sprint

### PR DESCRIPTION
**Pull request by josix**
Update August pycht sprint.

I've added some open issues to the agenda and also kept the last topics that are not discussed yet.

### CODE REVIEW

**Comment by yungyuc** [comment]
There is another sprint being planned 2 weeks after this one: https://github.com/sciwork/swportal/pull/75 .  There are two options: (i) keep this on 14th August, and (ii) combine with that one on 30th.

File: content/pages/sprint/2021/08early-pycht.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+Registration
+------------
+
+Please join `gather.town <https://gather.town/app/yLTe8mBDb8pogMOX/sciwork>`_
+on the day of sprint!!
+
+Date & time
+-----------
+
+14th August, Saturday, 2021, 14:00 -- 16:00
```
Comment (yungyuc) [inline content/pages/sprint/2021/08early-pycht.rst:49]: It's not early August but mid-August.

File: content/pages/sprint/2021/08early-pycht.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
+=================================================
+Scisprint 2021 Early August: Python ZH TW Doc
+=================================================
+
+:date: 2021-06-29 15:24
```
Comment (yungyuc) [inline content/pages/sprint/2021/08early-pycht.rst:5]: How about using a timestamp closer to this PR?

File: content/pages/sprint/2021/08early-pycht.rst [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+:save_as: sprint/2021/08early-pycht.html
+
+Scisprint 2021 Early August: Python ZH TW Doc
+=================================================
+
+This is the second sprint for for translating `Python official
+document <https://docs.python.org/3/>`__ from English to traditional Chinese
+(Taiwan; ZH TW locale).
+
+The agenda:
```
Comment (yungyuc) [inline content/pages/sprint/2021/08early-pycht.rst:16]: I am not sure we should use the same agenda as the previous sprint (https://sciwork.dev/sprint/2021/07early-pycht).  Perhaps it is good to have it updated?

File: content/pages/sprint/2021/08early-pycht.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+Date & time
+-----------
+
+14th August, Saturday, 2021, 14:00 -- 16:00
+
+Agenda
+------
+
+* **13:30 -- 14:00**: Gathering
```
Comment (yungyuc) [inline content/pages/sprint/2021/08early-pycht.rst:54]: It would also be good to move the agenda, if it is updated, here.

**Review by yungyuc — CHANGES_REQUESTED** [review]

**Comment by josix** [comment]
> There is another sprint being planned 2 weeks after this one: #75 . There are two options: (i) keep this on 14th August, and (ii) combine with that one on 30th.

I'm open to it. The benefit of this is we can have more participants which are perhaps interested in our works but we should discuss the agenda for running both sprints. Or we could held by ourself on 14th August. What do @yungyuc and @mattwang44's think about it?

**Comment by yungyuc** [comment]
The end of 

> > There is another sprint being planned 2 weeks after this one: #75 . There are two options: (i) keep this on 14th August, and (ii) combine with that one on 30th.
> 
> I'm open to it. The benefit of this is we can have more participants which are perhaps interested in our works but we should discuss the agenda for running both sprints. Or we could held by ourself on 14th August. What do @yungyuc and @mattwang44's think about it?

I am OK either way, too.  The August sprint is just published: https://sciwork.dev/sprint/2021/08, for your information.

File: content/pages/sprint/2021/08early-pycht.rst (hunk 3 above)
Comment (josix) [reply content/pages/sprint/2021/08early-pycht.rst:16]: Yeah. I've added and updated the description about rescheduling the going of the project and the review workflow. But I also keep some same topics that we have't discussed too much, I guess we could have more discussion about that in the next sprint.

File: content/pages/sprint/2021/08early-pycht.rst (hunk 4 above)
Comment (josix) [reply content/pages/sprint/2021/08early-pycht.rst:54]: Thanks for the notification! I'll add some detail items here.

File: content/pages/sprint/2021/08early-pycht.rst (hunk 3 above)
Comment (yungyuc) [reply content/pages/sprint/2021/08early-pycht.rst:16]: Sorry I overlooked the updates already made.

**Comment by yungyuc** [comment]
@josix any updates?

**Comment by josix** [comment]
Hello @yungyuc Thanks for the notification, I've updated the detail in the agenda and also correct the title and date of this sprint. Please have a look, thank you!

**Comment by yungyuc** [comment]
Thanks.  Could you rebase and squash to remove the pages from the other commit?

**Review by yungyuc — CHANGES_REQUESTED** [review]
Thanks. Could you rebase and squash to remove the pages from the other commit?

**Comment by josix** [comment]
Hello @yungyuc, I've removed the files that created from others commit, pleas have a check, thanks!

File: content/pages/sprint/index.rst [hunk 5]
```diff
@@ -13,6 +13,8 @@ hosted by sciwork.
 Scisprint 2021
 ==============
 
+* `Scisprint 2021 Mid August: Python ZH TW Doc <{filename}2021/08mid-pycht.rst>`__
+* `Scisprint 2021 August <{filename}2021/08.rst>`__
```
Comment (yungyuc) [inline content/pages/sprint/index.rst:17]: It's strange the line was not in your base, but it's ok.

File: content/pages/sprint/index.rst [hunk 6]
```diff
@@ -13,6 +13,8 @@ hosted by sciwork.
 Scisprint 2021
 ==============
 
+* `Scisprint 2021 Mid August: Python ZH TW Doc <{filename}2021/08mid-pycht.rst>`__
```
Comment (yungyuc) [inline content/pages/sprint/index.rst:16]: The sprints are ordered reversed chronically.  Could you move it lower than the August entry?

You can rename the other one to be Late August (but keep the URL unchanged).  But I think this is optional.

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: content/pages/sprint/index.rst (hunk 5 above)
Comment (josix) [reply content/pages/sprint/index.rst:17]: Probably I dropped  that commit when rebasing. Let me check for it.

File: content/pages/sprint/index.rst (hunk 6 above)
Comment (josix) [reply content/pages/sprint/index.rst:17]: Fixed it, I've rename it to "Late August" and also updated the order.

**Review by yungyuc — APPROVED** [review]

**Comment by yungyuc** [comment]
Thanks @josix .  I am merging this PR.  If you want to set up a sign-up page like https://github.com/sciwork/swportal/wiki/2021-August-sprint-sign-up-page , feel free to file an update PR.

**Comment by yungyuc** [comment]
Sorry, I just saw the conflict at https://github.com/sciwork/swportal/pull/77/conflicts .  @josix can you resolve it before I merge?

**Comment by josix** [comment]
> Sorry, I just saw the conflict at https://github.com/sciwork/swportal/pull/77/conflicts . @josix can you resolve it before I merge?

Got it. I've resolved it.

**Comment by yungyuc** [comment]
> > Sorry, I just saw the conflict at https://github.com/sciwork/swportal/pull/77/conflicts . @josix can you resolve it before I merge?
> 
> Got it. I've resolved it.

Thanks a lot!

**Pull request merged**